example/test.js

```diff
@@ -8,14 +8,42 @@ console.log();
 
 const content = fs.readFileSync('./dist/main.bundle.js');
 if (
-  content.toString()
-    .indexOf(`(typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {}).SENTRY_RELEASE = {
+  content
+    .toString()
+    .indexOf(
+      `var _global = typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};`
+    ) !== -1
+) {
+  console.log('Saul Goodman, found "var _global" assignment in bundle');
+} else {
+  console.error('Boom, did not find "var _global" assignment in bundle');
+  process.exit(1);
+}
+
+if (
+  content.toString().indexOf(
+    `_global.SENTRY_RELEASE = {
   id: "foo"
-}`) !== -1
+};`
+  ) !== -1
 ) {
   console.log('Saul Goodman, found SENTRY_RELEASE in bundle');
-  process.exit(0);
```

Comment:
Why remove this `process.exit`, and why not add it to the previous check? Should we expect to find all the assignments? If we successfully find one but fail the others, should we exit? If this is the case, should we move the `process.exit(0)` in the last block to run after it?